react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+
+export function StopWatch() {
+  const [seconds, setSeconds] = useState(0);
+  const [play, setPlay] = useState(false);
+  const [pauseID, setPauseID] = useState(null);
+
+  function startTimer() {
+    if (!play) {
+      const count = setInterval(() => {
+        setSeconds((seconds) => seconds + 1);
+      }, 1000);
+      setPlay(true);
+      setPauseID(count);
+    }
+    clearInterval(pauseID);
+  }
+
+  function playPause() {
+    setPlay(!play);
+  }
+
+  function controlButton() {
```

Comment:
Though it's optional, I prefer to just define a variable that holds this value, instead of putting it inside of a function.